AndelaOSP/art-dashboard
# Ft admin can assign asset to user #159022592

## What does this PR do?
It enables an admin to allocate an asset.

## Description of Task to be completed?
Given An existing user 
And an existing available asset
When I select the user to assign it to
And I click save
Then the asset should be assigned to that user

## How should this be manually tested?
On the assets page, click on view icon for any asset, this loads the asset details. 
On the description tab, on your far right, you will see the implemented assign asset functionality.

## What are the relevant pivotal tracker stories?
#159022592

## Any background context you want to add?
N/A

## Important notes
N/A

## Packages installed
N/A

## Deployment note
N/A

## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
## Screenshots
<img width="540" alt="screen shot 2018-07-20 at 13 44 18" src="https://user-images.githubusercontent.com/27014080/42998550-247af536-8c23-11e8-9904-033e398767eb.png">
